src/components/calcite-action-group/calcite-action-group.tsx

```diff
@@ -24,6 +25,16 @@ export class CalciteActionGroup {
    */
   @Prop({ reflect: true }) expanded = false;
 
+  /**
+   * Indicates the horizontal, vertical, or grid layout of the component.
+   */
+  @Prop({ reflect: true}) layout?: Layout = "vertical";
```

Comment:
nitpick: The property is optional but has a default value so the optional isn't needed.

Reply:
Ah...good catch.